src/wp-includes/class-wp-scripts.php

```diff
@@ -390,9 +409,16 @@ public function do_item( $handle, $group = false ) {
 			return true;
 		}
 
-		$tag  = $translations . $cond_before . $before_handle;
-		$tag .= sprintf( "<script%s src='%s' id='%s-js'></script>\n", $this->type_attr, $src, esc_attr( $handle ) );
-		$tag .= $after_handle . $cond_after;
+		$tag  = $translations . $cond_before . $before_script;
+		$tag .= sprintf(
+			"<script%s src='%s' id='%s-js'%s%s></script>\n",
+			$this->type_attr,
+			esc_url( $src ),
```

Comment:
Being the devil's advocate here, I wonder why the original code here didn't `esc_url()`. Maybe intentional? Of course in principle I'd agree that we should escape here, but since the old code didn't do it, there's arguably some small unnecessary risk in changing it now which also is unrelated to what this PR does overall.

Reply:
Great point. I lean towards being cautious in the sake of security here initially but something to watch out for and follow up on during the beta period.

Reply:
Out of curiosity I started looking into the history here and then realized that this value is already being escaped above, so no need to double escape. Removed redundant escaping and added an inline doc in [78bb58f](https://github.com/WordPress/wordpress-develop/pull/4391/commits/78bb58fa654744d74416e61bdd4c2614d75bb490).